internal/attrs/attrs.go

```diff
@@ -42,6 +42,7 @@ type Attributes struct {
 	IPv4Len uint8  // Prefix length for IPv4.
 	IPv6Len uint8  // Prefix length for IPv6.
 	MTU     uint16
+	ID      uint32 // p4rt interface id
```

Comment:
I think I would prefer named something like P4ID or P4InterfaceID.  ID is so generic it opens the field up to misuse.

Reply:
It's a good suggestion, but if we change it here, it makes it different from the OC path.  Maybe I should add a comment with the full path name?

This ID maps directly to the OC path [/interfaces/interface/config/id](https://openconfig.net/projects/models/schemadocs/yangdoc/openconfig-interfaces.html#interfaces-interface-config-id)

Reply:
Added comment with the full path to disambiguate